<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="dynamixel_groupbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>691</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Dynamixel</string>
    </property>
    <widget class="QGroupBox" name="baudrate_groupbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>661</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>BaudRate</string>
     </property>
     <widget class="QPushButton" name="baudrate_button_57600">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>57600</string>
      </property>
     </widget>
     <widget class="QPushButton" name="baudrate_button_115200">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>115200</string>
      </property>
     </widget>
     <widget class="QPushButton" name="baudrate_button_1000000">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>1000000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="baudrate_button_2000000">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>2000000</string>
      </property>
     </widget>
     <widget class="QPushButton" name="baudrate_button_3000000">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>111</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>3000000</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="command_groupbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>661</width>
       <height>141</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Command</string>
     </property>
     <widget class="QPushButton" name="read_write_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bulk_read_write_button">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Bulk Read/Write</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="id_1_groupbox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>ID 1 </string>
     </property>
     <widget class="QLineEdit" name="id_1_position_lineedit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_1_velocity_lineedit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>75</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Velocity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="id_2_groupbox">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>340</y>
       <width>331</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="title">
      <string>ID 2 </string>
     </property>
     <widget class="QLineEdit" name="id_2_position_lineedit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_2_velocity_lineedit">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>72</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Velocity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Position</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
